merlin-sdk/src/main/java/gov/nasa/jpl/aerie/merlin/protocol/types/TaskReturnValue.java

```diff
@@ -0,0 +1,3 @@
+package gov.nasa.jpl.aerie.merlin.protocol.types;
+
+public record TaskReturnValue<T>(T returnValue) {}
```

Comment:
`TaskReturnValue<T>` seems to be equivalent to `T` itself. Is there a particular reason we want a wrapper here?

Reply:
Hmm... I originally had a separate `Map<TaskId, TaskReturnValue<?>> taskReturns` map, and I didn't like the look of a `Map<TaskId, Object>`. However, now TaskReturnValue is folded into ExecutionState, so maybe it isn't adding much anymore.

Reply:
TaskReturnValue has been eliminated :+1: